apps/nova/src/components/Plans/Content.tsx

```diff
@@ -0,0 +1,210 @@
+import * as Tabs from "@radix-ui/react-tabs";
+import React, { useCallback, useMemo, useState } from "react";
+import { Typography } from "@litespace/luna/Typography";
+import { PlanCard } from "@litespace/luna/PlanCard";
+import { AnimatePresence, motion } from "framer-motion";
+import CurvedDashedLine from "@litespace/assets/CurvedDashedLine";
+import cn from "classnames";
+import { useToast } from "@litespace/luna/Toast";
+import { LocalId } from "@litespace/luna/locales";
+import { useFormatMessage } from "@litespace/luna/hooks/intl";
+import { PlansDataProps, Tab } from "@/components/Plans/types";
+
+const Content: React.FC<{
+  plans?: PlansDataProps["data"] | null;
+  error?: string | null;
+}> = ({ plans, error }) => {
+  const intl = useFormatMessage();
+  const toast = useToast();
+  const [tab, setTab] = useState<Tab>("monthly");
+  const tabs = useMemo((): Array<{ value: Tab; label: LocalId }> => {
+    return [
+      {
+        value: "monthly",
+        label: "plans.labels.monthly",
+      },
+      {
+        value: "quarter",
+        label: "plans.labels.quarter",
+      },
+      {
+        value: "half",
+        label: "plans.labels.half",
+      },
+      {
+        value: "annual",
+        label: "plans.labels.annual",
+      },
+    ];
+  }, []);
+
+  const notifyComingSoon = useCallback(() => {
+    toast.success({
+      title: intl("plans.subscribe.soon.title"),
+      description: intl("plans.subscribe.soon.description"),
+    });
+  }, [intl, toast]);
+
+  const makeTabPlans = useMemo(
+    () => (tab: Tab) =>
+      (
+        <Animate key={tab} tab={tab}>
+          <div className="flex gap-6 mt-[124px]">
+            {plans?.[tab].map((plan, idx) => (
+              <div key={plan.id} className={cn({ "-mt-10": idx === 1 })}>
+                <PlanCard
```

Comment:
Let's give a plan a max width of 368px (like in the design) and center all 3 cards in the middle. No need to make it scratched.